src/LanguageNegotiationSetter.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace Drupal\oe_multilingual;
+
+use Drupal\Core\Config\ConfigFactoryInterface;
+use Drupal\Core\Language\LanguageInterface;
+
+/**
+ * Helper service for language negotiation method configuration.
+ */
+class LanguageNegotiationSetter {
```

Comment:
Let's provide an interface for this service so that it can be overridden by custom implementations while still ensuring that the API stays the same.

Reply:
Great, there is an interface now, thanks a lot! :+1: